<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManipFinderForm</class>
 <widget class="QWidget" name="ManipFinderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Rank</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Duelist</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>From duel start?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Desired drop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Starting seed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="duelistBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="rankBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="dropBox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="seedBox"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="startCheck">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="dumpsBtn">
       <property name="text">
        <string>Add dumps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fusionBtn">
       <property name="text">
        <string>Add fusions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="equipsBtn">
       <property name="text">
        <string>Add equips</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fieldsBtn">
       <property name="text">
        <string>Add field spell</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="battleBtn">
       <property name="text">
        <string>Add battle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="directBtn">
       <property name="text">
        <string>Add direct attack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trapBtn">
       <property name="text">
        <string>Add trap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OptsChanger" name="optsChanger">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>225</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>248</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="manipBtn">
       <property name="text">
        <string>Find manip</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choices</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Seed value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="manipBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Seed index</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListView" name="choicesBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fastest manip</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="convertLeftBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="convertRightBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OptsChanger</class>
   <extends>QScrollArea</extends>
   <header>optschanger.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
